docs/project-model/project-json-reference.md

```diff
@@ -0,0 +1,494 @@
+# Project.json reference
+
+* [name](#name)
+* [version](#version)
+* [description](#description)
+* [summary](#summary)
+* [copyright](#copyright)
+* [title](#title)
+* [entryPoint](#entryPoint)
+* [projectUrl](#projectUrl)
+* [licenseUrl](#licenseUrl)
+* [iconUrl](#iconUrl)
+* [compilerName](#compilerName)
+* [testRunner](#testRunner)
+* [authors](#authors)
+* [owners](#owners)
+* [tags](#tags)
+* [language](#language)
+* [releaseNotes](#releaseNotes)
+* [requireLicenseAcceptance](#requireLicenseAcceptance)
+* [embedInteropTypes](#embedInteropTypes)
+* [compile](#compile)
+* [content](#content)
+* [resource](#resource)
+* [preprocess](#preprocess)
+* [publishExclude](#publishExclude)
+* [shared](#shared)
+* [namedResource](#namedResource)
+* [packInclude](#packInclude)
+* [exclude](#exclude)
+* [contentBuiltIn](#contentBuiltIn)
+* [compileBuiltIn](#compileBuiltIn)
+* [resourceBuiltIn](#resourceBuiltIn)
+* [excludeBuiltIn](#excludeBuiltIn)
+* [dependencies](#dependencies)
+* [tools](#tools)
+* [commands](#commands)
+* [scripts](#scripts)
+* [compilationOptions](#compilationOptions)
+    * [define](#define)
+    * [nowarn](#nowarn)
+    * [additionalArguments](#additionalArguments)
+    * [languageVersion](#languageVersion)
+    * [allowUnsafe](#allowUnsafe)
+    * [platform](#platform)
+    * [warningsAsErrors](#warningsAsErrors)
+    * [optimize](#optimize)
+    * [keyFile](#keyFile)
+    * [delaySign](#delaySign)
+    * [publicSign](#publicSign)
+    * [emitEntryPoint](#emitEntryPoint)
+    * [xmlDoc](#xmlDoc)
+    * [preserveCompilationContext](#preserveCompilationContext)
+* [analyzerOptions](#analyzerOptions)
+    * [languageId](#languageId)
+* [configurations](#configurations)
+* [frameworks](#frameworks)
+    * [dependencies](#dependencies)
+    * [frameworkAssemblies](#frameworkAssemblies)
+    * [wrappedProject](#wrappedProject)
+    * [imports](#imports)
+    * [bin](#bin)
+        * [assembly](#assembly)
+
+<a name="name"></a>
+## name
+Type: String
+
+The name of the project, this is also the name of the (nuget) package the project will produce.
+
+<a name="version"></a>
+## version
+Type: String
+
+The semver version of the project, also used for the nuget package.
+
+<a name="description"></a>
+## description
+Type: String
+
+A longer decsiption of the project.
+
+<a name="summary"></a>
+## summary
+Type: String
+
+A short description of the project.
+
+<a name="copyright"></a>
+## copyright
+Type: String
+
+The copyright information for the project.
+
+<a name="title"></a>
+## title
+Type: String
+
+The friendly name of the project, can contain spaces and special characters not allowed in the name.
```

Comment:
> can contain spaces and special characters not allowed in the name.

When you say "name", you mean the `name` attribute, yes?

We'll also have to get info on what uses this.